frontend/src/components/assessment-creation/CreateAssessementModal.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Button } from "@chakra-ui/react";
+import { PlusOutlineIcon } from "../../assets/icons";
+
+const CreateAssessementModel = (): React.ReactElement => {
+  return (
+    <>
+      <Button rightIcon={<PlusOutlineIcon />} variant="primary" my={2}>
+        Create Assessment
+      </Button>
+    </>
+  );
+};
+
+export default CreateAssessementModel;
```

Comment:
If you look at the designs, there isn't a `CreateAssessmentModal`, just a new page. So this button should just link to the route of the page that Cindy is working on.